ee/api/test/test_dashboard_collaborators.py

```diff
@@ -34,9 +34,11 @@ def test_list_collaborators_as_person_without_edit_access(self):
         response = self.client.get(
             f"/api/projects/{self.test_dashboard.team_id}/dashboards/{self.test_dashboard.id}/collaborators/"
         )
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
         response_data = response.json()
+        response_data = sorted(response_data, key=lambda entry: entry["user"]["email"])
```

Comment:
The test was flaky: https://github.com/PostHog/posthog/actions/runs/9535772415/job/26281928310?pr=22995

Reply:
Great catch!